tilavarauspalvelu/models/recurring_reservation/model.py

```diff
@@ -27,9 +27,9 @@
 
 
 class RecurringReservation(models.Model):
+    ext_id: uuid.UUID = models.UUIDField(default=uuid.uuid4, editable=False, unique=True)  # ID for external systems
```

Comment:
I think `public_uuid` (or just `uuid` or something similar) would fit better to reflect that it's an uuid field, not a "normal" id field, and that the uuid does not come from an external source, but rather that it's shared to the public

Reply:
`uuid` creates too many name collisions by itself (`import uuid` -> type field as `uuid.UUID`).  I think `ext_id` is fine since it means "external ID" = "ID used for external systems". `public_uuid` IMO has the same possibility for misunderstanding where the id comes from, but I did add a comment for that purpose (`# ID for external systems`)

Reply:
And it's not like our regular IDs aren't "public", they just aren't as reliable for external systems to query with since they might change e.g. if the database is recreated.

Reply:
Changed the name to `ext_uuid` to reflect that it is a UUID.